src/core/data-cite.js

```diff
@@ -52,6 +52,10 @@ function requestLookup(conf) {
     if (frag) {
       href = new URL(frag, href).href;
     }
+    if (frag == null && path == null) {
```

Comment:
I'm not sure if `== null` is the right check here (`null` is generally used to tell that's it's explicitly set to null).
```suggestion
    else if (!path) {
```
Though `if (!frag && !path)` might be preferable as it's more explicit.